collect_app/src/main/java/org/odk/collect/android/receivers/NetworkReceiver.java

```diff
@@ -94,9 +105,13 @@ private boolean isFormAutoSendOptionEnabled(NetworkInfo currentNetworkInfo) {
     }
 
     /**
-     * @param isFormAutoSendOptionEnabled represents whether the auto-send option is enabled at the app level
+     * If the app-level autosend setting is enabled, send all filled forms that don't specify not to
+     * autosend at the form level. If the app-level autosend setting is disabled, send all filled
+     * forms that specify to send at the form level.
+     *
+     * @param isAutoSendAppSettingEnabled whether the auto-send option is enabled at the app level
```

Comment:
This is a bit nitpicky, but I'm noticing that "autosend" is sometimes capitalized like it's one word (`autosend`, `KEY_AUTOSEND`) and sometimes capitalized like it's two words (`autoSend` or `AutoSend`).  Since in prose it's mostly written as one word ("autosend") and only occasionally hyphenated ("auto-send") and never as two words ("auto send"), I think it would be ideal to capitalize it always as one word.